modules/setting/repository.go

```diff
@@ -78,6 +78,7 @@ var (
 			DefaultMergeMessageOfficialApproversOnly bool
 			PopulateSquashCommentWithCommitMessages  bool
 			AddCoCommitterTrailers                   bool
+			RetargetChildsOnClose                    bool
```

Comment:
```suggestion
			RetargetChildrenOnClose                    bool
```

Reply:
Furthermore, wouldn't it be
```suggestion
			RetargetChildsOnMerge                    bool
```
?